### PR TITLE
Add SectionTabs component

### PR DESCRIPTION
**Pull request by iAdramelk**
Desktop:

image

Mobile:

image

Will add storybook page later.

### CODE REVIEW

File: components/SectionTabs/SectionTabs.tsx [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+const SectionTabsItem = ({
+  children,
+  description,
+  id,
+  selected,
+  src,
+  title,
+  onChange,
+}: SectionTabsItemProps) => {
+  const onClick = useCallback(() => onChange(id), [id]);
```
Comment (alex-kovoy) [inline components/SectionTabs/SectionTabs.tsx:35]: lets stop using `useCallback` as it does more harm than good when applied by default. Here is a good article on the subject 
 https://dmitripavlutin.com/dont-overuse-react-usecallback/
 
 it's OK to re-evaluate below function on this type of re-render.
 
```ts
const onClick = () => onChange(id);
```

File: components/SectionTabs/SectionTabs.tsx [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+interface SectionTabsProps {
+  children: React.ReactNode;
+  subtitle: string;
+}
+
+export const SectionTabs = ({ children, subtitle }: SectionTabsProps) => {
+  const [currentTab, setCurrentTab] = useState<number>(0);
+  const onChange = useCallback((index: number) => setCurrentTab(index), []);
+
+  const childTabProps = useMemo(
```
Comment (alex-kovoy) [inline components/SectionTabs/SectionTabs.tsx:99]: we do not need `useMemo` here as well.

File: components/SectionTabs/SectionTabs.tsx [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+};
+
+interface SectionTabsProps {
+  children: React.ReactNode;
+  subtitle: string;
+}
+
+export const SectionTabs = ({ children, subtitle }: SectionTabsProps) => {
+  const [currentTab, setCurrentTab] = useState<number>(0);
+  const onChange = useCallback((index: number) => setCurrentTab(index), []);
```
Comment (alex-kovoy) [inline components/SectionTabs/SectionTabs.tsx:97]: lets remove another `useCallback`

**Review by alex-kovoy — APPROVED** [review]
lets clean up the code by removing useMemo and useCallback and then good to merge.

File: components/SectionTabs/SectionTabs.tsx (hunk 1 above)
Comment (iAdramelk) [reply components/SectionTabs/SectionTabs.tsx:35]: Fair point. Will remove them here. Price of replacing event listener in DOM here should be roughly the same.

**Pull request merged**